joomla/joomla-cms
# Support non-english chars in JS

The previous code doesn't really support non-english characters and for example fails on german umlauts. This code is used in com_finder, so when you are searching for something like "Europäer", you wont get any highlighting for the searched term in the search results.

### How to test
1. Enable Smart Search content plugin
2. Edit/create an article with the word "Europäer"
3. Search for that word in the frontend
4. See that the word "Europäer" is not highlighted in the search results
5. Apply patch
6. Search again and now see it being highlighted.
